### PR TITLE
add verbose flag

### PR DESCRIPTION
**Pull request by vitorsantanna2**
I took advantage of @cherryramatisdev code to use args instead of stdin, and increased the -v flag, to print the command being tested at the end of each iteration.

### CODE REVIEW

File: src/run.cr [hunk 1]
```diff
@@ -23,7 +23,14 @@ module Run
...
 
-  def self.run_test(exercise_index : Int, file_name : String, run_with_stdin = true)
+ def self.get_command(initial_command : JSON::Any, file_name : String) : String
+    initial_command = initial_command.to_s
+
+    return initial_command.sub("$", file_name) if initial_command.includes?("$")
+    return "#{initial_command} #{file_name}"
+  end
+
+  def self.run_test(exercise_index : Int, file_name : String, run_with_stdin = true, run_with_verbose = true)
```
Comment (cherryramatisdev) [inline src/run.cr]: I don’t think verbose should be enabled by default

**Review by cherryramatisdev — CHANGES_REQUESTED** [review]
Great work! Thanks for looking into my issue

File: src/run.cr (hunk 1 above)
Comment (vitorsantanna2) [reply src/run.cr]: Thanks for the suggestion, I changed the default value

**Review by cherryramatisdev — APPROVED** [review]
All good from me

@lanjoni

**Review by lanjoni — APPROVED** [review]
Awesome! I'm just going to resolve the conflict that is being pointed out and I'll merge! Thank you very much for your contribution!

**Pull request merged**